tests/foundationals/fuyu/test_fuyu.py

```diff
@@ -0,0 +1,81 @@
+import random
+from pathlib import Path
+from warnings import warn
+
+import pytest
+import torch
+from PIL import Image
+from torch import device as Device
+from torchvision.transforms.functional import (  # type: ignore[reportMissingTypeStubs]
+    to_pil_image,  # type: ignore[reportUnknownVariableType]
+)
+from transformers import FuyuForCausalLM, FuyuProcessor  # type: ignore[reportMissingTypeStubs]
+
+from refiners.fluxion.utils import load_from_safetensors, manual_seed, no_grad
+from refiners.foundationals.fuyu.fuyu import Fuyu, Fuyu8b, create_fuyu
+
+
+@pytest.fixture(scope="module")
+def our_model(test_weights_path: Path, test_device: Device) -> Fuyu:
+    weights = test_weights_path / f"fuyu8b.safetensors"
+
+    if not weights.is_file():
+        warn(f"could not find weights at {weights}, skipping")
+        pytest.skip(allow_module_level=True)
+
+    config = Fuyu8b().with_device(test_device)
+    model = create_fuyu(config)
+    tensors = load_from_safetensors(weights)
+    model.load_state_dict(tensors)
+    return model
+
+
+@pytest.fixture(scope="module")
+def ref_processor() -> FuyuProcessor:
+    processor: FuyuProcessor = FuyuProcessor.from_pretrained(pretrained_model_name_or_path="adept/fuyu-8b")  # type: ignore[reportUnknownMemberType, reportAssignmentType]
+    return processor
+
+
+@pytest.fixture(scope="module")
+def ref_model(test_device: Device) -> FuyuForCausalLM:
+    model: FuyuForCausalLM = FuyuForCausalLM.from_pretrained(pretrained_model_name_or_path="adept/fuyu-8b").to(  # type: ignore
+        device=test_device
+    )
+    return model  # type: ignore[reportUnknownVariableType]
+
```

Comment:
nit, you can probably simplify a bit the code here
```suggestion
@pytest.fixture(scope="module")
def ref_processor(test_device: torch.device) -> FuyuProcessor:
    return FuyuProcessor.from_pretrained(  # type: ignore
        pretrained_model_name_or_path="adept/fuyu-8b",
        device=test_device,
    )


@pytest.fixture(scope="module")
def ref_model(test_device: Device) -> FuyuForCausalLM:
    return FuyuForCausalLM.from_pretrained(  # type: ignore
        pretrained_model_name_or_path="adept/fuyu-8b",
        device=test_device,
    )

```

Reply:
Indeed directly returning simplifies the code, however using the "device_map" argument for from_pretained requires accelerate. 

"Using `low_cpu_mem_usage=True` or a `device_map` requires Accelerate: `pip install accelerate"

Reply:
oh you're right, I missed that since I had accelerate installed.